chore:updated Readme (#8)
Blackcipher101/Shopping-Cart: README.md

<h1>Shopping cart</h1>

> <p>This is created Pyhon's Django Framework</p>

Reviewer: ```suggestion
<p>This is created Python's Django Framework</p>
```

<p>This program will do the following</p>
 <ul style="list-style-type:disc;">
  <li>Signup</li>

>  <p>Sign up uses buitin Userauth libarary in django it takes username and password and send its primary checks it csrf token for security </p>

Reviewer: ```suggestion
 <p>Sign up uses builtin Userauth library in Django it takes username and password and sends the primary checks. It also uses csrf token for security. </p>
```

<img src="img/screen1.png">

 
 
 
 
 
 
 
 
 
 
 
  <li>Login</li>

>   <p>Login uses buitin Userauth libarary in django it takes username and password and send its primary checks it also uses csrf token</p>

Reviewer: ```suggestion
  <p>Login uses builtin Userauth libarary in Django. It takes username and password and sends the primary checks. It also uses csrf token</p>
```

  <img src="img/sceen3.png">
 
  <li>Display Products</li>

>   <p>I created model known as product with its follwoing attributes the built a specific url and sent requset to view product list

Reviewer: ```suggestion
  <p>I created a model known as product with its following attributes the built a specific url and sent request to view the product list.
```

>  and renderd it into template</p>

Reviewer: ```suggestion
 and rendered it into template</p>
```

 <img src="img/sceen4.png">
   <li>Display Details</li>

>  <p>The link diplays all the details of the product in  separate page </p>

Reviewer: ```suggestion
 <p>The link displays all the details of the product in a separate page </p>
```

 <img src="img/sceen5.png">
 
   
  <li>Add to a cart</li>

>   <p>created models order and order items when add to cart is pressed it request a view acces the product id and add it to order item then its added to order</p>

Reviewer: ```suggestion
  <p>Created models order and order items when add to cart is pressed. It requests a view access to the product id and adds it to order item after which it is added to order</p>
```

  <img src="img/sceen6.png">
 <img src="img/sceen7.png">
  
  <li>Calculate total</li>

>   <p>the add to cart view returns product price tottal</p>

Reviewer: ```suggestion
  <p>The add to cart view returns total product price.</p>
```

  <li>Delete item</li>
  <p>It calls delete view which deletes the order item by using its object id </p>
  <img src="img/sceen8.png">
 <img src="img/sceen9.png">
  <li>Add inventory</li>

>  <p>I made a custom form and django form template to add new product</p>

Reviewer: ```suggestion
 <p>I made a custom form and Django form template to add new product</p>
```
